spec/system/admin/products_v3/products_spec.rb

```diff
@@ -53,54 +53,64 @@
       within ".pagination" do
         click_link "2"
       end
+
+      expect(page).to have_content "Showing 16 to 30"
       expect_page_to_be 2
       expect_per_page_to_be 15
       expect_products_count_to_be 15
     end
 
     it "can change the number of products per page" do
       select "50", from: "per_page"
+
+      expect(page).to have_content "Showing 1 to 50"
       expect_page_to_be 1
       expect_per_page_to_be 50
       expect_products_count_to_be 50
     end
   end
 
   describe "search" do
-    before do
-      visit admin_products_v3_index_url
-    end
-
     context "product has searchable term" do
       # create a product with a name that can be searched
       let!(:product_by_name) { create(:simple_product, name: "searchable product") }
 
+      before do
+        visit admin_products_v3_index_url
+      end
+
       it "can search for a product" do
         search_for "searchable product"
 
         expect(page).to have_field "search_term", with: "searchable product"
+        # expect(page).to have_content "1 product found for your search criteria."
```

Comment:
Looks good, but why did you comment these out?

Reply:
Oh oops, these should have been marked as pending.